language/go/resolve.go

```diff
@@ -166,16 +166,16 @@ func resolveGo(c *config.Config, ix *resolve.RuleIndex, rc *repo.RemoteCache, r
 		return label.New("bazel_gazelle", pkg, "go_default_library"), nil
 	}
 
+	if gc.depMode == externalMode {
+		return resolveExternal(rc, imp)
```

Comment:
I can also fall back to current behavior if external dependency resolution fails for some reason. Let me know.